### DISCUSSION TITLE
Click-to-enlarge for images in GitHub Blog

### DISCUSSION THREAD

**Discussion by rovo89** [community/community#72067] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

The official GitHub blog is quite interesting to learn about new features. The posts often contain images demonstration the changes - but unfortunately, they're displayed too small to see the details. `Right-click -> Open image in new tab` works, but it would be nice if we could just click on the image to see it in full size (some kind of lightbox).

**Comment by fokrul511** [comment]
this is right
